docs/reference/migration/migrate_5_4.asciidoc

```diff
@@ -0,0 +1,8 @@
+[[breaking-changes-5.4]]
+== Breaking changes in 5.4
+
+[[breaking_54_delete_by_query_changes]]
+[float]
+=== Delete By Query API changes
```

Comment:
I wasn't sure about this either. In the 5.1 doc, it has explicitly `Indices API changes`, but in the 5.3 doc, it has `REST API changes`. I also think that the change does not impact only the REST layer. Users creating DBQ requests through Java, for example, will also have to deal with this.

Having said that, I don't have enough experience on how this is best documented.

Final call?